### ISSUE TITLE
Training PoinTr on Custom Dataset

### ISSUE THREAD

**Issue by kaali-billi** [yuxumin/PoinTr#138]
Hi I am trying to fine-tune/ train a PoinTr/AdaPoinTr model on my custom dataset containing ¬4000 point clouds, It is a complex 3D structure. Can you help me understand the basic size of a dataset to get good reconstruction performance. The complete point cloud is 13,300 points and the inputs are in the range 2000-8000 points. Currently I am training with the given PoinTr.yaml file and training parameters.

**Comment by kaali-billi** [comment]
![scale_problem](https://github.com/yuxumin/PoinTr/assets/61189809/a7efeb35-9950-4f42-9363-6d6960b41fc3)
Red Point Cloud - Ground Truth
Green PC - Input
Multi-colored_pc - AdaPoinTr Output. 

my question is why is the output Point cloud bigger than the input and GT in scale , while giving good shape reconstruction accuracy ?

**Comment by kaali-billi** [comment]
The scaling issue becomes greater as the Input point cloud is moved away from the origin [0,0,0], 
![scale_prblm](https://github.com/yuxumin/PoinTr/assets/61189809/c9df8114-bf70-4c13-b703-52b8175da4ad)

The green/black dot shows the camera position / origin

**Comment by kaali-billi** [comment]
@yuxumin please let me know if this is inherent to the model itself, the scaling and translation issue during reconstruction, I have tried training on normalized and de_nomralized data with no improvement.  The current dataset size is (6600,500,500) Training , test, val split

**Comment by rahulsai1219** [comment]
@kaali-billi I am also trying to train with my custom dataset that has 16000 points in the complete point cloud and 8000 to 12000 points in the input clouds. Could you please share the PoinTr.yaml and training parameters that worked best worked for you? I am having serious performance issues and I am finding it challenging to debug what might be going wrong.
